config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
```

Comment:
I would put them separate for logical separation.

And for now (and probably upcoming weeks) it would be nice if we could trigger the job independent of the other job -- we don't want bugs and experimentation here to mess up with PRs from other people.